backend/corpora/peaceportal/peaceportal.py

```diff
@@ -0,0 +1,513 @@
+import os
+import os.path as op
+import logging
+from datetime import datetime
+from langdetect import detect
+from langdetect.lang_detect_exception import LangDetectException
+
+from django.conf import settings
+
+from addcorpus.corpus import ParentCorpusDefinition, FieldDefinition
+from addcorpus.es_mappings import int_mapping, keyword_mapping, main_content_mapping, text_mapping
+from addcorpus.es_settings import es_settings
+from addcorpus.extract import Constant
+from addcorpus.filters import MultipleChoiceFilter, RangeFilter
+
+class PeacePortal(ParentCorpusDefinition):
+    '''
+    Base class for corpora in the PEACE portal.
+
+    This supplies the frontend with the information it needs.
+    Child corpora should only provide extractors for each field.
+    Consequently, create indices (with alias 'peaceportal') from
+    the corpora specific definitions, and point the application
+    to this base corpus.
+    '''
+
+    title = "PEACE Portal"
+    description = "A collection of inscriptions on Jewish burial sites"
+    # store min_year as int, since datetime does not support BCE dates
+    min_year = -530
```

Comment:
Does this work in the corpus overview?

Reply:
It's only used as a minimal value for the range slider, so yes. Will investigate whether a different date library can address this problem as part of #1292 .